mitodl/ocw-studio
# use new theme assets pipeline definition

# What are the relevant tickets?
https://github.com/mitodl/ocw-studio/issues/1928

# Description (What does it do?)
This PR replaces the use of the static `theme-assets-pipeline.yml` definition with the new `ThemeAssetsPipelineDefinition` object based pipeline.

# How can this be tested?
 - Spin up `ocw-studio`, making sure you have Minio and a custom Github org configured
 - Run `docker-compose exec web ./manage.py upsert_theme_assets_pipeline`
 - Browse to your local Concourse instance at http://localhost:8080 and log in with test / test
 - Find the theme assets pipeline, unpause it if it is paused and trigger a build
 - Once the build is complete, inspect the `upload-theme-assets-task` output and verify that the build output is going to the right place
 - In the `ocw-studio` UI, publish any site and verify that the JS / CSS output from the theme assets build you just triggered is being used
